# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pingus/boost-mt.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./SConstruct.orig     2009-06-15 09:26:11 UTC (rev 3997)
+++ ./SConstruct     2009-08-03 22:20:46 UTC (rev 3998)
@@ -457,7 +457,8 @@
                                     'src/input/xinput_device.cpp']

     if not config.CheckLibWithHeader('boost_signals', 'boost/signals.hpp', 'c++'):
-        fatal_error += "  * library 'boost_signals' not found\n"
+       if not config.CheckLibWithHeader('boost_signals-mt', 'boost/signals.hpp', 'c++'):
+          fatal_error += "  * library 'boost_signals' not found\n"
 
     if not config.CheckLibWithHeader('png', 'png.h', 'c++'):
        fatal_error += "  * library 'png' not found\n"

